string name
---
float64 mass
